client/packages/invoices/src/OutboundShipment/DetailView/Toolbar.tsx

```diff
@@ -46,8 +48,8 @@ export const Toolbar: FC = () => {
                   <CustomerSearchInput
                     disabled={isDisabled}
                     value={otherParty}
-                    onChange={otherParty => {
-                      update({ otherParty });
+                    onChange={async ({ id: otherPartyId }) => {
+                      await updateName({ id, otherPartyId });
```

Comment:
First I was wondering if it is a good idea to update the name in onChange and there shouldn't be a confirmation dialog? But I guess its fine as it is.